### ISSUE TITLE
No such file or directory when importing images in code

### ISSUE THREAD

**Issue by AlessandroFano** [Nuitka/Nuitka#2244]
Hello,

I am developing a tool in python with tkinter.
The structure of my project is the following:

 - project folder:
    - src
       - main.py
       - ...other files.py
       - changelog.md
    - res
       - img
         - images used by my project

In my main code I import several images in the following way:
`    question_icon = PhotoImage(file = "../res/img/question_icon_32.png")
`

When I try to build a one file executable with nuitka, if I execute it the \src folder then everything works, but if I try to move it to some other folder then I get this error when importing the images:

couldn't open "../res/img/question_icon_32.png": no such file or directory.

I figure the problem is the way I import the images, but I wasn't able to understand exactly what am I doing wrong.
I have python 3.11 and nuitka 1.5.8
I call nuitka with:
`python -m nuitka --onefile --onefile-tempdir-spec="%TEMP%\onefile_%PID%_%TIME%" -o myprogram.exe --enable-plugin=tk-inter --include-data-dir="C:\...project_folder\res"=res --include-data-file="C:\...project_folder\src\changelog.md"=changelog.md C:\...project_folder\src\main.py`

of course "C:\...project_folder" is the full absolute path to the project folder.

Do you have any suggestion what am I doing wrong?

**Comment by kayhayen** [comment]
In the template, I have made it clear, that not filling out the issue template is not acceptable. I am closing this kind of report as invalid. The main reason is that I do not want to be left guessing. For you it's easy to erase the issue template and expect me to ask you questions only as necessary, but I cannot do that. I would much rather wait for you to create a new report, fill out the template, or wait for somebody with the necessary skill to do it.

In short: No issue template, no fun. Therefore, I am likely just deleting the issue, as it serves as a bad example and detraction for searches.  And even worse, it might keep good reporters from reporting as is this "already known."

Also consider reading the User Manual.
